site/docs/v1/tech/apis/_application-request-body.adoc

```diff
@@ -255,12 +260,18 @@ endif::includeRoles[]
 [field]#application.samlv2Configuration.audience# [type]#[String]# [optional]#Optional# [default]#defaults to `issuer`# [since]#Available since 1.6.0#::
 The audience for the SAML response sent to back to the service provider from FusionAuth. Some service providers require different audience values than the `issuer` and this configuration option lets you change the `audience` in the response.
 
-[field]#application.samlv2Configuration.callbackURL# [type]#[String]# [required]#Required# [since]#Available since 1.6.0#::
+[field]#application.samlv2Configuration.authorizedRedirectURLs# [type]#[Array<String>]# [optional]#Optional# [since]#Available since 1.20.0#::
+A list of URLs that are authorized for FusionAuth to redirect the user after logging in via SAML.
+
+[field]#application.samlv2Configuration.callbackURL# [type]#[String]# [optional]#Optional# [since]#Available since 1.6.0# [deprecated]#Deprecated#::
 The URL of the callback (sometimes called the Assertion Consumer Service or ACS). This is where FusionAuth sends the browser after the user logs in via SAML.
 
 [field]#application.samlv2Configuration.debug# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`# [since]#Available since 1.6.0#::
 Whether or not FusionAuth will log SAML debug messages to the event log. This is useful for debugging purposes.
 
+[field]#application.samlv2Configuration.defaultVerificationKeyId# [type]#[UUID]# [optional]#Optional#  [since]#Available since 1.20.0#::
+Default verification key to use for HTTP Redirect Bindings, and for POST Bindings when no key is found in request. Required if `requiredSignedRequests` is set to `true`
```

Comment:
When we reference a field in the description you should use this style: `[field]#requireSignedRequests#`.